src/modals/modal-overlay/modal-overlay.component.js

```diff
@@ -103,9 +109,12 @@ angular.module('patternfly.modals').component('pfModalOverlayContent', {
 
       ctrl.ok = function (label, actionFn) {
         if (typeof actionFn === "function") {
-          actionFn();
+          if (actionFn() !== false) {
```

Comment:
Only concern would be an existing application returning false but not expecting the modal to remain open. Unlikely unless it is just happenstance. I think it's OK to assume that unlikelyhood is great enough to proceed.

Reply:
Yeah, I tend to agree that scenario is unlikely.  If an app was returning `false`, I'm not sure why or what controller would be processing a true or false?  The component executes the button actionFns and it wasn't expecting any return values.